mirrord/layer/src/lib.rs

```diff
@@ -388,6 +388,15 @@ fn set_globals(config: &LayerConfig) {
             .try_into()
             .expect("Failed setting up outgoing traffic filter!");
 
+        if outgoing_selector.is_remote() && !REMOTE_DNS.get().unwrap() {
+            warn!(
```

Comment:
This wouldn't show up on IDEs, would it?
Asking because of @eyalb181's comment at the end of https://github.com/metalbear-co/mirrord/issues/1768#issue-1838260835.

Maybe move this to `MirrordExecution::start` where you have `progress` and show it as a progress message?

Reply:
Oh, that's new to me.

Do we add it in both places, or just in the `MirrordExecution` part?

Reply:
Definitely only in the CLI (btw, I think `set_globals` is called on each `execve`, so with many programs, e.g. npm we would get this log multiple times if it were in `set_globals`). Emitting both the progress message and a log from the same place is ok, but I think just the progress message is enough.

Reply:
Moved.